btle/ecies/README.md

```diff
@@ -0,0 +1,66 @@
+# BTLE ECIES Example
+
+This example demonstrates a lightweight method for exchanging data securely over any type of publicly visible link.
+
+The first phase is key establishment, which is done through ECDH and HDKF. ECC was chosen for these examples because its lightweight and widely used. 
+Then salt exchanged for each message to prevent reply attacks. The encryption is done with AES CBC. The data integrity is done using HMAC-SHA256.
+
+The peer's key should be validated against a known key or certificate. Client should hash and verify this public key against trusted certificate (already exchanged) out of band. An ECC signature is about 65 bytes.
+
+## ECC Encrypt/Decrypt Example
+
+See `BTLESecureMessageExchange.pdf` for details.
+
+
+### Building
+
+The wolfSSL library must be built and installed using:
+
+```
+./configure --enable-ecc --enable-eccencrypt --enable-hkdf
```

Comment:
I found that `--enable-ecc` and `--enable-hkdf` were enabled and didn't strictly need to be specified. Also it maybe of interest that ECIES will work with `--enable-cryptonly` but makes sense that these can't be disabled.